ajax/libs/dna.js/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "dna.js",
+  "npmName": "dna-js",
```

Comment:
There are only 5 releases on NPM but 37 on GitHub, so let's use git auto-update config instead of npm auto-update. You can take a look at [autoupdate.md](https://github.com/cdnjs/cdnjs/blob/master/documents/autoupdate.md).
BTW, could you please also fill npm information in pull request template? Thanks. ref: https://www.npmjs.com/package/dna.js